website/docs/plugin/which-sdk.mdx

```diff
@@ -48,12 +48,8 @@ should consider using the framework if you would benefit from the ability to:
 ## Do You Need Any Features That the Framework Doesn't Provide (Yet)?
```

Comment:
```suggestion
## Do You Need Any Features That the Framework Does Not Provide (Yet)?
```

Suggest changing "Doesn't" to "Does Not" to avoid using contractions.